pkgs/by-name/co/coc-diagnostic/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  mkYarnPackage,
+  fetchYarnDeps,
+  fetchFromGitHub,
+  nix-update-script
+}: mkYarnPackage rec {
```

Comment:
Please do not use mkYarnPackage, even though there is no hook for yarn yet unfortunately.
see also, https://github.com/NixOS/nixpkgs/issues/324246

Reply:
I will be glad to fix the derivation once #318015 is merged, so that I can properly test and build my changes.

Reply:
Added the hooks, this should be good now.